documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,147 @@
+
+. <<New_Project,Create a New Maven Project>>
+. <<New_Module,Create a New Maven Module>>
+. <<Maven_Support,Add Maven Support to an Existing Non-Maven Project>>
+
+Maven projects serve as parents for Maven modules. For each Maven module attached to a Maven project, a section is added to the project’s pom.xml file. Each command run by the Maven project is run against each of the included modules first.
+
+[[New_Project]]
+== 1. Create a New Maven Project
+The first step when using Maven with an IDE is to create a Maven project that can contain multiple independent Maven modules. Follow the instructions to create a Maven project:
+
+. Create a new project:
+.. In the workspace, navigate to menu:File[New > Other].
+.. Type `maven` in the `Filter` field and select `Maven Project`.
+.. Click `Next` to continue.
+. In the first step of the `New Maven Project` wizard, enter the initial project details:
+.. Select the `Use default Workspace location` check box to use the default workspace. If required, specify a non-default location by deselecting the check box and using the `Browse` button to specify the desired target directory.
+.. The following optional steps are available for further configuration, if required:
+... (Optional) Select the `Create a simple project (skip archetype selection)` check box. If this check box is selected, the `Select an Archetype` step of the wizard is skipped.
+... (Optional) Select the `Add project(s) to working set` check box to add additional projects to the working set.
+... (Optional) Click `Advanced` to view additional optional advanced configuration for the new Maven project, such as whether to `Resolve Workspace projects`, `Profiles`, and any `Name templates` to set a desired naming convention for the project.
++
+.Create a New Maven Project
+image::mvnbasic_project_name_location.png[Create a New Maven Project]
++
+.. When the configuration is complete, click `Next` to continue.
+. In the second step of the `New Maven Project` wizard, enter the archetype details:
+.. Ensure that the `Show the last version of Archetype only` check box is selected. This ensures that only the latest version of each archetype displays.
+.. Select an archetype based on the purpose of the project you are creating. Use the keyword `maven-archetype-quickstart` in the `Filter` field to locate a sample Maven project archetype.
++
+.Select a Project Archetype
+image::mvnbasic_select_archetype.png[Select a Project Archetype]
++
+.. Click `Next` to continue.
+. In the third step of the `New Maven Project` wizard, specify the archetype parameters:
+.. Enter the desired group ID in the `Group Id` field.
+.. Enter the desired artifact ID in the `Artifact Id` field.
+.. Set the `Version` field’s value to `0.0.1-SNAPSHOT` (or any other unique value).
```

Comment:
no, it cannot just be any unique value.

Maven has rules for how a version value looks like and there are also recommendations.

Plus, -SNAPSHOT has a very specific meaning. 

see details at http://books.sonatype.com/mvnref-book/reference/pom-relationships-sect-pom-syntax.html